rclpy/src/rclpy/_rclpy_qos_event.c

```diff
@@ -32,13 +32,18 @@ typedef PyObject * (* _qos_event_data_filler_function)(_qos_event_callback_data_
 
 static
 bool
-_check_rcl_return(rcl_ret_t ret, const char * error_msg)
+_check_rcl_return(rclpy_module_state_t * module_state, rcl_ret_t ret, const char * error_msg)
 {
+  if (!module_state) {
+    PyErr_Format(PyExc_RuntimeError, "_check_rcl_return got NULL module state");
+    return NULL;
```

Comment:
Should this be `return false`?

Reply:
`return false` in 02a795e2f9222524f8637d81087741a0d58f6150